addons/survey/views/survey_templates_management.xml

```diff
@@ -118,12 +118,18 @@
                             </div>
                             <div class="row">
                                 <div class="col-12 col-md-4 offset-md-4 text-center">
-                                    <input id="session_code" type="text" placeholder="e.g. 4812" autocomplete="off"
-                                           class="form-control o_survey_question_text_box font-weight-bold bg-transparent text-primary text-center rounded-0 p-2 w-100"/>
+                                    <input id="session_code" type="text" placeholder="e.g. 4812" autocomplete="off" class="form-control o_survey_question_text_box font-weight-bold bg-transparent text-primary text-center rounded-0 p-2 w-100" t-attf-value="{{session_code if session_code else ''}}"/>
```

Comment:
Would split this line in 2 to avoid having a very long line (we could have "class" on its own line by itself for example).